BLE_GattServer_FOTAService/mbed_app.json

```diff
@@ -0,0 +1,18 @@
+{
+    "target_overrides": {
+        "*": {
+            "platform.stdio-convert-newlines": true,
+            "platform.stdio-baud-rate": 115200,
+            "cordio.desired-att-mtu": 200,
+            "cordio-ll.max-acl-size": 204,
+            "cordio.rx-acl-buffer-size": 204,
```

Comment:
These two flags are specific to NRF, can you move them there ? 
Even if it isn't working on Disco yet that would be nice if at least it build so CI passes.

Reply:
Sort of. Depending on the value of `cordio.rx-acl-buffer-size`, the system will reserve statically more space for the ACL buffers in the host part of the stack. The value chosen will depends on what the controller supports. In this case, if `cordio-ll.max-acl-size` is set to `160` then  it makes sense to set `cordio.rx-acl-buffer-size` to the same value. 

On other controllers, the value will change so an override per target is desirable.